MRFIXIT: this patch adds 8bitdo to the controllers that use SDL2 instead of UDEV.

DISABLED because of the new patch which ALWAYS uses SDL2 to avoid a UDEV bug with USB hubs

--- a/configgen/generators/libretro/libretroControllers.py
+++ b/configgen/generators/libretro/libretroControllers.py
@@ -161,4 +161,4 @@
     return 'udev'
 
 # list of controllers that works only with sdl2 driver
-sdl2driverControllers = ['Bluetooth Wireless Controller', 'szmy-power', 'XiaoMi Bluetooth Wireless GameController', 'ipega Bluetooth Gamepad', 'ipega Bluetooth Gamepad   ', 'Moga Pro 2 HID']
+sdl2driverControllers = ['Bluetooth Wireless Controller', 'szmy-power', 'XiaoMi Bluetooth Wireless GameController', 'ipega Bluetooth Gamepad', 'ipega Bluetooth Gamepad   ', 'Moga Pro 2 HID', '8Bitdo NES30 Pro']
